keps/sig-storage/695-skip-permission-change/README.md

```diff
@@ -171,9 +171,10 @@ type PodSecurityContext struct {
         - Metric name: storage_operation_errors_total{operation_name=volume_mount}
         - [Optional] Aggregation method: cumulative counter
         - Components exposing the metric: kubelet
-    - volume ownerhip change timing mtrics: We are also going to add metrics that track time it takes for volume ownerhip change to happen. We will update this section with the name of metrics.
-
-
+    - volume ownerhip change timing mtrics:
+        - Metric name: storage_operation_duration_seconds{operation_name=volume_fsgroup_recursive_apply}
```

Comment:
question on the operation name. would "volume_fsgroup_apply" be a better name because the function can also skip doing the recursive processing?

Reply:
Yeah we could that. I can file an issue for figuring out the metric label.